### PR TITLE
Everbloom v2

### PR DESCRIPTION
**Pull request by sheerryy**
Changes:

- Remove Edition
- Move artwork metadata to contract level
- implement FLIP-0636 - NFT Metadata
- Make Artwork struct and update metadata

### CODE REVIEW

File: contracts/EverbloomV2.cdc [hunk 1]
```diff
@@ -0,0 +1,974 @@
+// SPDX-License-Identifier: MIT
```
Comment (sheerryy) [inline contracts/EverbloomV2.cdc]: @SiriusDely this File requires review

File: tests/src/everbloom-v2.js [hunk 2]
```diff
@@ -0,0 +1,338 @@
+import {
```
Comment (sheerryy) [inline tests/src/everbloom-v2.js:1]: @SiriusDely this file requires review

File: tests/test/everbloom-v2/everbloom-v2.test.js [hunk 3]
```diff
@@ -0,0 +1,278 @@
+import path from "path";
```
Comment (sheerryy) [inline tests/test/everbloom-v2/everbloom-v2.test.js:1]: @SiriusDely this file requires review

**Review by SiriusDely — APPROVED** [review]
Awesome!!!

**Pull request merged**